workspaces/kubernetes-native-dxp/skaffold.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: skaffold/v2beta28
+kind: Config
+build:
+  artifacts:
+  - image: k8s-native-dxp
+    custom:
+      buildCommand: ./gradlew resolve buildDockerImage -Pdocker.image.id=$IMAGE
+      dependencies:
+        paths:
+        - modules/**/*
+  - image: remote-app-a
+    custom:
+      buildCommand: lxc-extensions/liferay-hello-world/build.sh
+      dependencies:
+        paths:
+        - lxc-extensions/liferay-hello-world/public
+        - lxc-extensions/liferay-hello-world/src
+        - lxc-extensions/liferay-hello-world/package.json
+  - image: custom-rest-service
+    custom:
+      buildCommand: lxc-extensions/custom-rest-service/build.sh
+      dependencies:
+        paths:
+        - lxc-extensions/custom-rest-service/src/**/*
+        - lxc-extensions/custom-rest-service/pom.xml
+deploy:
+  kustomize:
+    paths:
+    - k8s/dxp/
+    - k8s/remoteappa/
+    - k8s/custom-rest-service/
+portForward:
```

Comment:
I really like this scaffold concept, but can we try and see if we can forgot the port-forwards? I guess we need to see if we can get what I have working in your MAC

Reply:
So for me, even with minikube, i have to port-forward at least once to the ingress controller.  I have not found a way to get minikube to open up network access without at least one port-forward.  i would love it if we could avoid that though.

Reply:
Were you able to try the skaffold? where it monitors the files and redeploys the images automatically?

Reply:
I haven't tried the scaffold yet since I had lots of changes on PKCE and TLS but I really like the idea behind what I see and I like the idea of monitoring projects, that's just perfect.

w.r.t. to Ingress problems, have you tried the solution described [here](https://stackoverflow.com/questions/63388065/minikube-kubernetes-wont-allow-ingress-on-mac-despite-running-as-a-vm)?

Reply:
I'll try the ingress solution you mentioned.  Regarindg your changes, should I resend these after your changes?  I may be easier for me to merge them.

Reply:
> I haven't tried the scaffold yet since I had lots of changes on PKCE and TLS but I really like the idea behind what I see and I like the idea of monitoring projects, that's just perfect.
> 
> w.r.t. to Ingress problems, have you tried the solution described [here](https://stackoverflow.com/questions/63388065/minikube-kubernetes-wont-allow-ingress-on-mac-despite-running-as-a-vm)?

I'll give this a try, nothing has exactly right just yet.

Reply:
Even after using --vm=true and -driver=hyperkit i still have to have the port-forward.  But I just need the one port-forward created by skaffold now, not manually managing them like before.